Makefile

```diff
@@ -188,6 +188,9 @@ endif
 	docker push $(BOOTNODE_IMAGE_NAME):latest
 	docker push $(DOCKER_IMAGE_NAME):latest
 
+protoc-install:
+	apt install -y protobuf-compiler
```

Comment:
We should move this to a script in `_assets/scripts` so that we can deal more easily with multiple host platforms.